action.yml

```diff
@@ -0,0 +1,37 @@
+name: Add pull request to project
+description: GitHub Action to add new pull requests to the Community Contributions GitHub project
+
+runs:
+  using: composite
+  steps:
+
+    - name: Check if should be added
+      id: check-if-should-add
+      env:
+        PR_AUTHOR: ${{ github.event.pull_request.user.login }}
+      run: |
+        # We don't want CMS Squad member PRs to clutter the project board
+        echo "Author is $PR_AUTHOR"
+        for squad_member in 'GuySartorelli' 'emteknetnz' 'maxime-rainville'; do
+          if [[ $PR_AUTHOR == $squad_member ]]; then
+            echo "Author is in CMS Squad. Skipping."
+            echo "should_add_to_project=false" >> $GITHUB_OUTPUT
+            exit 0
+          fi
+        done
+        echo "should_add_to_project=true" >> $GITHUB_OUTPUT
+
+    - name: Generate token
+      id: generate-token
+      if: steps.check-if-should-add.outputs.should_add_to_project == 'true' && github.repository_owner == 'silverstripe'
+      uses: actions/create-github-app-token@v1
+      with:
+        app-id: ${{ vars.PROJECT_MANAGER_APP_ID }}
+        private-key: ${{ secrets.PROJECT_MANAGER_PRIVATE_KEY }}
```

Comment:
I'd like to standardise all the naming so that it's obvious what things relate to one another

I've renamed the app to "Silverstripe Community Contribute" (I would have used Contributions, though that exceeds the character limit, so that it's more obvious that the app relates to the particular github project

Could you update the name of these variables to:

COMMUNITY_CONTRIBUTIONS_APP_ID
COMMUNITY_CONTRIBUTIONS_APP_PRIVATE_KEY (or COMMUNITY_CONTRIBUTIONS_PRIVATE_KEY if that's more technically correct)

I'd prefer "Community board" over "Community Contributions" (easier to say, fits the character limit), though I'm not overly fussed

Reply:
Done